src/wazuh_db/wdb_shared.c

```diff
@@ -25,6 +25,29 @@ const char* WDBC_RESULT[] = {
     [WDBC_UNKNOWN] = "unk"
 };
 
+/// Array of component strings.
+const char* wdbc_valid_components[] = { 
+    "syscollector_processes",
```

Comment:
Considering that DBSync/RSync can be used by `syscheck`, this the list of components that trigger `wdb_parse_syscheck()` and end up managing `state`, `integrity_check_` and `integrity_clear` commands

<details><summary>Details</summary>
<p>

![2024-11-14_22-21](https://github.com/user-attachments/assets/630b6a63-e25d-4544-9ac8-e0db0c886778)

![2024-11-14_22-18](https://github.com/user-attachments/assets/7e8c0a93-4002-4e73-ab92-41b9e2cb8a27)

</p>
</details> 

The same for `syscollector`

<details><summary>Details</summary>
<p>

![2024-11-14_22-23](https://github.com/user-attachments/assets/dd702878-c318-4081-a06c-10967c9b19c4)


</p>
</details>

Reply:
I made an upgrade of the list, please tell me if it's ok. Tomorrow we can sync to verify if I understand your images correctly

Reply:
Ok , thank you.